lemur/constants.py

```diff
@@ -38,6 +39,9 @@
     "ECCSECT571R2",
 ]
 
+# For commonly reused regexes used by plugins
+EMAIL_RE = re.compile('^([\w+-.%]+@[-\w.]+\.[A-Za-z]{2,4},?)+$')
```

Comment:
@jtschladen this regex compiles now! I swapped the literal hyphen from `[\w-.]` to `[-\w.]` since (I believe) the position is interchangeable in character ranges (ran a couple of sanity checks locally).

Reply:
Nice, sounds good to me.